config/routes.rb

```diff
@@ -185,14 +185,14 @@
   end
 
   # Help
-  get '/help/:group', :to => 'questions#index', :as => :help
+  get '/help/:group', :to => 'guides#faq', :as => :help
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.